### PR TITLE
feat: GitHub templates

### PR DESCRIPTION
**Pull request by DA-344**
## Brief Description

Adds GitHub templates.

## Detailed Description

Adds new pull requests GitHub template, it is based of discord.py's `.github/PULL_REQUEST_TEMPLATE.md`
file.

## Change Log

- Created `.github/PULL_REQUEST_TEMPLATE.md`
- Created `.github/workflows/md-to-wiki.yaml` to autopush the changes in `FEATURES.md` to the Wiki

## Checklist

- [ ] PR contents come from a trusted source, such as DDevs Docs PR, [unofficial documentation](https://docs.discord.sex/), Discord staff mention, or others.
- [x] Changed files formatting works as expected.
- [x] This is **not** a features code change.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+When creating a pull request you must fill the template, which contains the following:
+
+- Brief Description: A short description that describes what the pull request has changed.
+- Detailed Description: A detailed description of what the pull request has changed.
+- Change Log: A list of the changes you've made to the repository files.
+- Checklist: A checklist that has to be filled depending on what your PR does.
+
+You also must make sure to provide a source to the Discord API Documentation or discord.py PR or any other "trusted"
+source like a Discord Developer/Staff member, a well-known bot developer, a well-known library developer, etcetera. Or
+if it is in the [Unofficial Discord API Documentation](https://docs.discord.sex/), you can also add it.
```
Comment (Soheab) [inline .github/CONTRIBUTING.md]:
```suggestion
You must also include a source from the Discord API Documentation, a discord.py pull request, or any other reputable source such as a Discord Developer/Staff member, a well-known bot/library developer. Alternatively, if the information is available in the [Unofficial Discord API Documentation](https://docs.discord.sex/), feel free to reference it.
```
I asked copilot to rewrite it idk

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+<!-- Explain with as much detail as possible your PR -->
+
+## Change Log
+
+<!-- Here add the changes you've made as unordered list -->
+<!-- WHEN MAKING A PR, REMOVE THIS LINE
+- Change log 1
+- Change log 2
+WHEN MAKING A PR, REMOVE THIS LINE -->
```
Comment (Soheab) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- List the changes you made, does not have to be in any particular order -->
- Example log 1
- Example log 2
```
too much imo

File: README.md [hunk 3]
```diff
@@ -14,10 +14,7 @@ You will notice that each table has a status column, this column indicates the s
 
 ## Contributing
 
-If you want to contribute to this repository, you can do so by creating a pull request. You can add a new feature, update an existing one or fix a mistake. If you want to add a new feature, please make sure to provide a source to the Discord API documentation, PR or any other "trusted" source like a Discord Developer / Staff member, a well-known bot developer or a well-known library developer.
-Or if it's in the [Unofficial Discord API Documentation](https://docs.discord.sex/), you can also add it.
-
-Each feature is listed per category, matching the Discord API documentation and sorted alphabetically. If you want to add a new feature, please make sure to add it to the correct category and sort it alphabetically.
+To see how to contribute in this library, take a look at [CONTRIBUTING.md](/.github/CONTRIBUTING.md)
```
Comment (Soheab) [inline README.md]:
```suggestion
To see how to contribute in this repository, take a look at [CONTRIBUTING.md](/.github/CONTRIBUTING.md)
```

**Comment by DA-344** [comment]
PR is ready to be merged, any followups will be commited directly instead.

**Review by Soheab — APPROVED** [review]
Looks good to me!

**Pull request merged**